src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -44,12 +48,38 @@ class EmojiPickerMenu extends Component {
         // If more emojis are ever added to emojis.js this will need to be updated or things will break
         this.unfilteredHeaderIndices = [0, 33, 59, 87, 98, 120, 147];
 
+        // Toggles which keys the search input will listen to
+        // NOTE: these need to be instance members so we can
+        // reference the same event handlers in memory when removing them
+        this.keyToDefaultPreventer = {};
+        ['ArrowDown', 'ArrowLeft', 'ArrowRight', 'ArrowUp'].forEach((key) => {
+            this.keyToDefaultPreventer[key] = (keyBoardEvent) => {
+                if (keyBoardEvent.key === key) {
+                    keyBoardEvent.preventDefault();
+                }
+            };
+        });
+        this.toggleKeysOnSearchInput = (keysToIgnore = [], keysToAccept = []) => {
+            keysToIgnore.forEach((key) => {
+                this.searchInput.addEventListener('keydown', this.keyToDefaultPreventer[key]);
+            });
+            keysToAccept.forEach((key) => {
+                this.searchInput.removeEventListener('keydown', this.keyToDefaultPreventer[key]);
+            });
+        };
+
         this.filterEmojis = _.debounce(this.filterEmojis.bind(this), 300);
+        this.highlightAdjacentEmoji = this.highlightAdjacentEmoji.bind(this);
+        this.scrollToHighlightedIndex = this.scrollToHighlightedIndex.bind(this);
+        this.toggleArrowKeysOnSearchInput = this.toggleArrowKeysOnSearchInput.bind(this);
         this.renderItem = this.renderItem.bind(this);
 
         this.state = {
             filteredEmojis: emojis,
             headerIndices: this.unfilteredHeaderIndices,
+            highlightedIndex: -1,
+            currentScrollOffset: 0,
+            shouldDisablePointerEvents: false,
```

Comment:
Why is this one called `shouldDisablePointerEvents` and not `arePointerEventsDisabled`

Reply:
Hmmm that's a good point, since this is updated by the callback that actually disables pointer events. I guess because the boolean is used in actually disabling the pointer events, but it makes more sense to call it that.